spec/dummy/client/app/startup/clientRegistration.jsx

```diff
@@ -4,6 +4,7 @@ import 'es5-shim';
 import ReactOnRails from 'react-on-rails';
 
 import HelloWorld from '../components/HelloWorld';
+import HelloWorldReason from '../components/HelloWorldReason.bs';
```

Comment:
@justin808 as is, it won't recognize `HelloWorldReason.bs` component. 
```
clientRegistration.jsx?209f:1 Uncaught Error: Cannot find module "../components/HelloWorldReason.bs"
```

Reply:
This is just not how you integrate the bs into the JS. The JS code needs the compiled file to import.